sample-compose/app/src/main/java/co/nimblehq/sample/compose/di/modules/OkHttpClientModule.kt

```diff
@@ -16,12 +19,29 @@ private const val READ_TIME_OUT = 30L
 class OkHttpClientModule {
 
     @Provides
-    fun provideOkHttpClient() = OkHttpClient.Builder().apply {
+    fun provideOkHttpClient(
+        chuckerInterceptor: ChuckerInterceptor
+    ) = OkHttpClient.Builder().apply {
         if (BuildConfig.DEBUG) {
             addInterceptor(HttpLoggingInterceptor().apply {
                 level = HttpLoggingInterceptor.Level.BODY
             })
+            addInterceptor(chuckerInterceptor)
             readTimeout(READ_TIME_OUT, TimeUnit.SECONDS)
         }
     }.build()
+
+    @Provides
+    fun provideChuckerInterceptor(
+        @ApplicationContext context: Context
+    ): ChuckerInterceptor {
+        val chuckerCollector = ChuckerCollector(
+            context = context,
+            showNotification = true,
+            retentionPeriod = RetentionManager.Period.ONE_HOUR
+        )
+
+        return ChuckerInterceptor.Builder(context).collector(chuckerCollector)
+            .alwaysReadResponseBody(true).build()
```

Comment:
Sames goes for the `sample-xml` & `template` 🙏 

```suggestion
        return ChuckerInterceptor.Builder(context)
                   .collector(chuckerCollector)
                   .alwaysReadResponseBody(true)
                   .build()
```